packages/sources/uscpi-one/src/config.ts

```diff
@@ -0,0 +1,14 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
+export const NAME = 'USCPIONE'
```

Comment:
The adapter path is `uscpi-one`, which to me makes sense that the name would be `USCPI_ONE`.

Reply:
done